sdk/digitaltwins/azure-digitaltwins-core/autorest.md

```diff
@@ -27,7 +27,7 @@ input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/14fb403
 
 output-folder: "./"
 license-header: MICROSOFT_MIT_SMALL
-use: '@autorest/java@4.0.3'
+use: '@autorest/java@4.0.4'
```

Comment:
are we also regenerating using the new version?

Reply:
Nothing changes here, we are just upgrading java dependency